src/sass/partials/_game-section.scss

```diff
@@ -0,0 +1,276 @@
+@use "./variables" as *;
+
+.your-card-title {
+  margin: 5% 0;
+  width: 100%;
+  display: block;
+}
+
+.game-container-icon {
+  position: absolute;
+  bottom: 5%;
+  left: 48%;
+}
+
+// GAME CONTAINER --------------------------------
+.game-container {
+  min-height: 100vh;
+  height: 100%;
+  padding: 10% 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  z-index: 20;
+  border-radius: 0 0 15px 15px;
```

Comment:
personally, i'm not visually font of the rounder corners at bottoms of your sections, to me it reads more as a CSS mistake than an intentional design choice. But if you like it I don't think there is any harm in keeping it!

Reply:
Cool thanks! I was unsure about them, just thought it looked a bit like a deck of cards splayed out haha but i appreciate the feedback :)